README.md

```diff
@@ -47,158 +62,69 @@ just docker-build astria-sequencer debug
 
-Pass your token from above to the `--celestia-bearer-token` flag.
+To run unit tests:
 
 ```sh
-./target/release/astria-sequencer-relayer --celestia-endpoint=http://astria-celestia-jsonrpc-client-test.localdev.me:80/jsonrpc/ --celestia-bearer-token=<token-from-above-step> --validator-key-file=$HOME/.cometbft/config/priv_validator_key.json 
+cargo test
 ```
 
-If Celestia is not running, pass the `--disable-writing` flag (and omit the celestia flags):
+Note that the `astria-proto` generates its code by running tests (and verifying
+that nothing changed). In order for its tests to run you also need
+[Buf](https://buf.build/docs/installation/) installed.
 
-```sh
-./target/release/astria-sequencer-relayer --validator-key-file=$HOME/.cometbft/config/priv_validator_key.json --disable-writing
-```
-
-#### Build and start astria go-ethereum
+## Formatting
 
-```sh
-git clone https://github.com/astriaorg/go-ethereum.git
-cd go-ethereum
-make geth
-./build/bin/geth --datadir ~/.astriageth/ init genesis.json
-./build/bin/geth --datadir ~/.astriageth/ --http --http.port=8545 --ws --ws.port=8545 --networkid=1337 --http.corsdomain='*' --ws.origins='*' --grpc --grpc.addr=localhost --grpc.port 50051
-```
+This project uses [rustfmt](https://github.com/rust-lang/rustfmt) to format rust
+sources, [taplo](https://github.com/tamasfe/taplo) to format toml files, and
+[markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) for
+markdown.
 
-#### Start the conductor
+### Rust
 
 ```sh
-./target/release/astria-conductor
+# Install rustfmt
+rustup +nightly-2023-07-07 component add rustfmt
+# Run rustfmt
+cargo +nightly-2023-07-07 fmt --all
 ```
 
-If Celestia is not running, pass the `--disable-finalization` flag:
+### Toml
 
 ```sh
-./target/release/astria-conductor --disable-finalization
-```
-
-#### Sending a transaction to the EVM rollup
-
-Import the following private key into Metamask or other Ethereum wallet:
-```
-0a6996ccaca77a1d48633da20062ae051e11c75f3f561dfd2ac01b0c9c874662
-```
-
-Alternatively, in the go-ethereum step, you can add genesis account allocations inside `go-ethereum/genesis.json` (you will need to re-init geth if you've already initialized it).
-
-Navigate to the `localhost:8545` network. Then, you can send transactions as normal. If you check the logs, you can see them move through the stack from sequencer -> relayer -> DA -> conductor -> geth.
-
-#### Clean up local environment
+# Install for macOS
+$ brew install taplo
+# Install for Arch Linux
+$ sudo pacman -S taplo
 
-Stop all running processes and 
-```sh
-pkill astria-sequencer-relayer && pkill astria-conductor && pkill geth && pkill astria-sequencer && pkill cometbft
-rm -rf ~/.cometbft && rm -rf ~/.astriageth
-just delete-cluster
+# Run
+$ taplo format
 ```
 
-## Testing
+### Markdown
 
-To run unit tests:
 ```sh
-cargo test
-```
+# Install for macOS w/ homebrew
+brew install markdownlint-cli2
+# Install globally as node package
+npm install markdownlint-cli2 --global
```

Comment:
It's IMO bad practice to install executable globally from untrusted sources (like npm) as it requires root on Linux systems (unless you use something like volta that changes `{prefix}`). Let's not suggest it here and instead link to https://github.com/DavidAnson/markdownlint-cli2#install

Also because we do it elsewhere, please add that it can be installed from the arch `extra` repos with `sudo pacman -S markdownlint-cli2`